packages/tasit-account/src/TasitAccount.js

```diff
@@ -3,6 +3,10 @@ import "ethers/dist/shims.js";
 // shims aren't injected with package import
 import { ethers } from "ethers";
 
+// Account creates an ethres.js Wallet, should it create an Account object that encapsulates the wallet?
```

Comment:
```suggestion
// Account creates an ethers.js Wallet, should it create an Account object that encapsulates the wallet?
```

Reply:
@marcelomorgado Feel free to accept this

Reply:
Done

Reply:
Thanks! Resolving